integration_tests/support/commands.ts

```diff
@@ -164,6 +165,7 @@ Cypress.Commands.add('updateLongQuestionSetInductionToArriveOnCheckYourAnswers',
   // 'Has the prisoner worked before' is the next page. This is asked on the short question set so will have answers but
   // we will make a change to exercise the screen flow
   Page.verifyOnPage(WorkedBeforePage) //
+    .selectWorkedBefore(HasWorkedBeforeValue.YES)
```

Comment:
I'm not sure this is right. Whilst it works I think it's masking the actual problem.
As per the comment this should already have an answer, so you shouldn't need to select anything (which we didn't beforehand)

I think you've missed a couple of changes to the `hasWorkedBefore` property in the wiremock stubs.

Have a look here:
https://github.com/ministryofjustice/hmpps-education-and-work-plan-ui/blob/bdbddc8ec69302b074707808ab3ba6605596d141/integration_tests/mockApis/educationAndWorkPlanApi.ts#L906C14-L906C33
and
https://github.com/ministryofjustice/hmpps-education-and-work-plan-ui/blob/bdbddc8ec69302b074707808ab3ba6605596d141/integration_tests/mockApis/educationAndWorkPlanApi.ts#L504

I think if you change both of these from the boolean value to the corresponding enum, I think the test should pass without this change 🤞